spec/services/email_tool_sender_spec.rb

```diff
@@ -175,7 +175,8 @@ def expect_email_sender_to_be_called_with(params)
       expect(service.errors[:base]).to include(/targets information has recently changed/)
     end
 
-    it 'validates the presence of country' do
+    # Skipping since default country will be set as US within EmailToolSender
+    xit 'validates the presence of country' do
```

Comment:
Can you update/replace this spec to show that US is set when no country code has been passed?

Reply:
Have left this xit as it is but created a test in 'creating an action' context.